src/Raven.Server/Documents/Operations/GetReplicationConflictsOperation.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Net.Http;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Operations;
+using Raven.Client.Http;
+using Raven.Client.Json;
+using Raven.Server.Json;
+using Sparrow.Json;
+
+namespace Raven.Server.Documents.Operations
+{
+    internal class GetReplicationConflictsOperation : IMaintenanceOperation<BlittableArrayResult>
+    {
+        private readonly string _documentId;
+        private readonly long _etag;
+
+        public GetReplicationConflictsOperation(string documentId = null, long etag = default)
+        {
+            _documentId = documentId;
+            _etag = etag;
+        }
+
+        public RavenCommand<BlittableArrayResult> GetCommand(DocumentConventions conventions, JsonOperationContext context)
+        {
+            return new GetReplicationConflictsCommand(_documentId, _etag);
+        }
+
+        internal class GetReplicationConflictsCommand : RavenCommand<BlittableArrayResult>
+        {
+            private readonly string _documentId;
+            private readonly long _etag;
+
+            public override bool IsReadRequest => true;
+
+            public GetReplicationConflictsCommand(string documentId = null, long etag = default)
+            {
```

Comment:
you need also to pass `start` & `pageSize` here no?

Reply:
done

Reply:
rename `etag` to `start`

Reply:
done